### PR TITLE
39 give the ability to switch between deployed backend and localhost

### PR DESCRIPTION
**Pull request by ProEdge2**
# Pull Request Summary

Fixed the docker file to include all subdirectories in both the front end and backend
Added the ability to switch between the deployed backend and the localhost backend for development purposes

## Description
Modified docker file
Added dotenv to the front end to allow for environmental variables

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that aren't relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Creating tests
- [ ] Frontend/UI change

## Has This Change Been Tested?

- [ ] Yes
- [x] No

If "yes", please...

**Describe the tests that you ran to verify your changes: **
**Provide instructions so we can reproduce: **
**List any relevant details for your test configuration: **

### CODE REVIEW

**Comment by ProEdge2** [comment]
@Ajinkya-B I fixed that problem we discussed over discord with the data formatter, I did it in a separate branch it was a one line chance which I merged into this branch prior to the merge of development. So that change will also be included in this pull request just wanted to document it

**Comment by Ajinkya-B** [comment]
> @Ajinkya-B I fixed that problem we discussed over discord with the data formatter, I did it in a separate branch it was a one line chance which I merged into this branch prior to the merge of development. So that change will also be included in this pull request just wanted to document it

Ok perfect, I reviewed the files and they look good!

File: front-end/src/services/ProjectDataService.js [hunk 1]
```diff
@@ -1,27 +1,27 @@
-import http from "../http-common/http-common.projects";
+import http from "../http-common/http-common";
 
 
 // This class contains all the functions that make API calls and return info from the calls.
 class ProjectDataService {
     getAllProjects() {
-        return http.get('/');
+        return http.get('/api/v1/projects/');
```
Comment (chwawa) [inline front-end/src/services/ProjectDataService.js:7]: we can tackle this later, but we should make a separate http-common for projects and transcripts so we can avoid directly depending on this API.

**Review by chwawa — APPROVED** [review]
Nice work Marco!

File: front-end/src/services/ProjectDataService.js (hunk 1 above)
Comment (ProEdge2) [reply front-end/src/services/ProjectDataService.js:7]: Yep maybe make a different common file for each use case? we can discuss that

**Pull request merged**